administration/src/bp-modules/applications/VerificationsView.tsx

```diff
@@ -68,6 +73,24 @@ export const getStatus = (verification: Application['verifications'][number]) =>
   }
 }
 
+export const JulicaVerificationQuickIndicator = memo(() => {
```

Comment:
This should be put into a separate functional component and not exported here

Reply:
just to clarify: It should be put into a own file.

Reply:
I have pushed an update, please let me know if that's what you meant